stable/fluent-bit/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluent-bit
-version: 2.4.0
+version: 2.4.1
```

Comment:
This pr looks good to me, except its a feature addition not a bug fix so it should probably be 2.5.0.

Reply:
Alright, updated :)